app/components/file/report-drawer/va-reports/index.hbs

```diff
@@ -0,0 +1,60 @@
+{{!@glint-nocheck: perform, did-insert, will-destroy}}
+<div
+  {{did-insert this.initializeDrawer}}
+  {{will-destroy this.removeReportCounterObserver}}
```

Comment:
can we do this `constructor` and `willDestroy` methods

Reply:
Done